composer.json

```diff
@@ -9,7 +9,7 @@
     }],
     "require": {
         "silverstripe/framework": "^4.0@dev",
-        "colymba/gridfield-bulk-editing-tools": "~2.1"
+        "colymba/gridfield-bulk-editing-tools": "^3.0@dev"
```

Comment:
the requirement of ^3.0 pulls in 2.x? that's got to be a bug... What version of composer are you using?